src/main/java/org/jenkinsci/plugins/parallel_test_executor/TestCaseCentricFormat.java

```diff
@@ -0,0 +1,21 @@
+package org.jenkinsci.plugins.parallel_test_executor;
+
+public enum TestCaseCentricFormat  {
+    DISABLED ("Disabled"),
+    TCNAME ("Enabled - testCaseName"),
+    CLNAME_DOT_TCNAME ("Enabled - className.testCaseName");
+
+    private final String description;
+
+    TestCaseCentricFormat(String description) {
+        this.description = description;
+    }
+    
+    public String getDescription() {
+        return description;
+    }
+
+    static public boolean isEnabled(TestCaseCentricFormat format) {
+        return (format != null) && (format != TestCaseCentricFormat.DISABLED);

```

Comment:
I did this construct to try and work around (what I thought was) serialization compatibility issues. But I'm not really a Java developer, and this is my first take on trying to contribute to a plugin, so I'm lacking some knowledge in this area.

I thought that adding a variable of type TestCaseCentricFormat to ParallelTestExecutor would  make it take on a null value if an old version of of ParallelTestExecutor was de-serialized. Then it would be impossible to guarantee that the value is always non-null when used in the ParallelTestExecutor code. So by not using the TestCaseCentricFormat directly but always call isEnabled(TestCaseCentricFormat format) I could skip null checks everywhere else.

Any tips on how I can make sure format is always non-null when taking de-serialization of an older version into account (or perhaps I've just gotten things wrong and that is not an really an issue...)?